running_steps.md

```diff
@@ -0,0 +1,14 @@
+The basic steps to run the PHANGS version 2 script. The PHANGS_v2 scripts processing can be divided into 4 steps: staging, imaging, processing and derving the data products. The detailed description for these 4 steps is in README.md. Here is the detailed description of steps to use PHANGS v2 scripts to product images.  
+- Generate the key folder by running `python start_project.py`
+    - You will go through bunch of questions which are quite straightforward. 
```

Comment:
Where are these questions? Are they in `start_project.py` or do they get asked in the terminal where you run that script?